packages/code/src/Code/Code.spec.tsx

```diff
@@ -211,7 +198,6 @@ describe('packages/Code', () => {
               return renderCode({
                 isLoading: false,
                 language: languageOptions[0].displayName,
-                panel: <Panel />,
```

Comment:
I think the panel prop should be here since we're testing `with panel slot` on line 193. Was this the test that was failing because two copy buttons were rendering?

Reply:
No no all of the tests were passing

Reply:
I can add this back

Reply:
Added!